packages/search-service/src/search-sitemap/resolver.ts

```diff
@@ -0,0 +1,23 @@
+import { Request } from 'express';
+import { SiteMaps } from './schema';
+
+export const getSiteMap = async (req: Request) => {
+  const { contentType } = req.query;
+
+  const SiteMapdata = await (
```

Comment:
Use small camel case for variables. Reserve capital for Classes, Interfaces or Types.
```suggestion
  const sitemapData = await (
```